pkg/kubernetes/scripts/registry.js

```diff
@@ -163,6 +164,59 @@
                 return select().kind("Project");
             };
 
+            // TODO: invalidate cache on role changes
+            var sarCache = { };
```

Comment:
This should probably be combined with the other role caches that happen in policy